File: Backend/Models/Data/Patient.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace BuenDoctorAPI.Models.Data
{
    public partial class Patient
    {
        public Patient()
        {
            Appointments = new HashSet<Appointment>();
        }

        public string PatientId { get; set; }
        public string FirstName { get; set; }
        public string SecondName { get; set; }
        public string FirstSurname { get; set; }
        public string SecondSurname { get; set; }
        public byte[] ProfilePicture { get; set; }
        public DateTime Birthday { get; set; }
        public string Email { get; set; }
        public string Phone { get; set; }
        public string Mobile { get; set; }
        public string Direction { get; set; }
        public string Neighborhood { get; set; }
        public string City { get; set; }
        public bool? Status { get; set; }

        public virtual ICollection<Appointment> Appointments { get; set; }
    }
}

File: buen_doctor_app/lib/models/Patient.dart
import 'package:flutter/material.dart';

import 'Appointment.dart';

class Patient {
  late final String patientId;
  late final String firstName;
  late final String secondName;
  late final String firstSurname;
  late final String secondSurname;
  late final Image profilePicture;
  late final DateTime birthday;
  late final String email;
  late final String phone;
  late final String mobile;
  late final String direction;
  late final String neighborhood;
  late final String city;
  late final bool status;

  late final List<Appointment> appointments;
}
